src/memory-analyzer/analyze_symbol.h

```diff
@@ -145,6 +145,11 @@ class symbol_analyzert
     const exprt &zero_expr,
     const source_locationt &location);
 
+  exprt get_pointer_to_member_value(
```

Comment:
Needs documentation, especially w.r.t. to the struct+offset thing

Reply:
Done.